proto/zitadel/user/v2beta/user_service.proto

```diff
@@ -248,9 +248,6 @@ service UserService {
         permission: "user.write"
         org_field: "organization"
```

Comment:
removed, should be permission: "authenticated" right?

Reply:
ou yes... now only deleteUser needs to be changed as well (incl. checking permission in the command)

Reply:
ah totally missed that, I think I got all the other calls, also added the permission check everywhere, not quite sure for the AddHumanUser though if it will work there as expected.